gapic-generator/lib/gapic/presenters/method_rest_presenter.rb

```diff
@@ -116,10 +116,19 @@ def nonstandard_lro?
       # @return [Boolean]
       #
       def can_generate_rest?
-        (@main_method.kind == :normal) &&
+        (@main_method.kind == :normal || @main_method.kind == :server) &&
           http_bindings.first&.verb? &&
           http_bindings.first&.path?
       end
+
+      ##
+      # Whether this method is a server-streaming method
+      #
+      # @return [Boolean]
+      #
```

Comment:
Remove

Reply:
Actually, we need it. In `gapic-generator/templates/default/service/rest/client/method/def/_response.erb`, it checks `<%- elsif method.rest.server_streaming? -%>`. I could remove `.rest`, but all other `if` conditions use it, so prefer to leave it as is.